src/creators/__tests__/change-directory-error.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const expect = require('expect');
+const { isFSA } = require('flux-standard-action');
+const { keys, isEqual } = require('lodash');
+
+const { changeDirectoryError } = require('../');
+const { ERROR } = require('../../constants');
+const { CHANGE_DIRECTORY_FAILURE } = require('../../status-constants');
+
+
+describe(`changeDirectoryError creator`, function(){
+  const actionProperties = [
+    'type',
+    'payload'
+  ];
+  const payloadProperties = [
+    'notification',
+    'status',
+    'error'
+  ];
+  const dirpath = 'change/directory/error/';
+  const err = new Error('Change dir error');
+  const action = changeDirectoryError(dirpath, err);
+  const actionKeys = keys(action);
+  const payloadKeys = keys(action.payload);
+
+  it(`returns a 'Flux Standard Action'`, function(done){
+    expect(isFSA(action)).toEqual(true);
+    done();
+  });
+
+  it(`returns an action with a type equal to 'ERROR'`, function(done){
+    expect(action.type).toEqual(ERROR);
+    done();
+  });
+
+  it(`returns a payload.notification msg that includes the dirpath`, function(done){
+    expect(action.payload.notification).toInclude(dirpath);
+    done();
+  });
+
+  it(`returns a payload.status that matches 'CHANGE_DIRECTORY_FAILURE'`, function(done){
+    expect(action.payload.status).toEqual(CHANGE_DIRECTORY_FAILURE);
+    done();
+  });
+
+  it(`returns the original error in payload.error`, function(done){
+    expect(action.payload.error).toEqual(err);
+    done();
+  });
+
+  it(`returns an action object that only has ${actionProperties.length} known ${actionProperties.length === 1 ? 'property' : 'properties'}`, function(done){
+    expect(isEqual(actionKeys, actionProperties)).toEqual(true);
+    done();
+  });
+
+  it(`returns an action.payload object that only has ${payloadProperties.length} known ${payloadProperties.length === 1 ? 'property' : 'properties'}`, function(done){
+    expect(isEqual(payloadKeys, payloadProperties)).toEqual(true);
```

Comment:
this should probably just be 3 expects:
```js
expect(keys(action.payload)).toInclude('notification');
expect(keys(action.payload)).toInclude('status');
expect(keys(action.payload)).toInclude('error');
```
That can be DRYed up a bit, but it makes it more clear